sql_metadb/derived_tables/instance_administrative_notes.sql

```diff
@@ -0,0 +1,23 @@
+-- This derived table extracts administrative notes from the instance record.
+DROP TABLE IF EXISTS instance_administrative_notes;
+
+CREATE TABLE instance_administrative_notes AS 
+SELECT 
+	it.id AS instance_id,
+	it.hrid AS instance_hrid,
+	adminNotes.jsonb #>> '{}' AS administrative_notes,
+	adminNotes.ordinality AS administrative_notes_ordinality
+FROM 
+	folio_inventory.instance__t AS it
+	LEFT JOIN folio_inventory.instance AS inst ON inst.id = it.id 
+	CROSS JOIN LATERAL jsonb_array_elements(jsonb_extract_path(inst.jsonb, 'administrativeNotes')) WITH ORDINALITY AS adminNotes (jsonb);

```

Comment:
all-lowercase table names

```suggestion
	CROSS JOIN LATERAL jsonb_array_elements(jsonb_extract_path(inst.jsonb, 'administrativeNotes')) WITH ORDINALITY AS admin_notes (jsonb);
```